src/control/server/config_test.go

```diff
@@ -304,6 +305,12 @@ func TestServer_ConfigValidation(t *testing.T) {
 			},
 			FaultConfigBadControlPort,
 		},
+		"bad shutdown timeout value": {
+			func(c *Configuration) *Configuration {
+				return c.WithShutdownTimeout("123abc")
```

Comment:
Does "123abc" have any meaning?

Reply:
it's an invalid string specification that will not parse as a time.Duration and will therefore fail the config validation

Reply:
Ah, I didn't see this was in test code.